test/modules/workspace/utils/storageUtils.test.ts

```
@@ -0,0 +1,605 @@
+  it('should return empty array when Workspaces directory does not exist', async () => {
+    vi.mocked(exists).mockResolvedValue(false);
+
+    const result = await loadAllWorkspaces();
+
+    expect(result).toEqual([]);
+    expect(readDir).not.toHaveBeenCalled();
+  });
+
+  it('should handle missing updatedAt field', async () => {
+    const workspace = createMockWorkspaceData();
+    delete (workspace as any).updatedAt;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)